src/index.html

```diff
@@ -7,6 +7,342 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="MOYO logo"
+        class="logo"
+      >
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" data-qa="nav-hover">laptops & computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="cards">
+        <div class="cards__container">
+          <div class="cards__item card" data-qa="card">
+            <img
+              src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27"
+              class="card__image"
+            >
+
+            <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
don't use `h1` tag in card because there should be only one h1 title on the page